src/core/mac/mac_frame.cpp

```diff
@@ -769,104 +706,125 @@ void Frame::SetPayloadLength(uint16_t aLength)
 
 uint8_t Frame::SkipSecurityHeaderIndex(void) const
```

Comment:
Suggest to use the "Hide whitespace changes" option to look at the diff here and below.